lib/wallet/rpc.js

```diff
@@ -889,6 +890,28 @@ class RPC extends RPCBase {
     return null;
   }
 
+  async importName(args, help) {
+    if (help || args.length < 1 || args.length > 2) {
+      throw new RPCError(errs.MISC_ERROR,
+        'importpubkey "name" ( rescanHeight )');
+    }
+
+    const wallet = this.wallet;
+    const valid = new Validator(args);
+    const name = valid.str(0);
+    const rescan = valid.u32(1);
```

Comment:
Let's call this variable `height`. Earlier in the code (`importPubkey`) we use `rescan` as a boolean indicating whether to rescan the wallet.